### PR TITLE
Fix tx fee options

### PR DESCRIPTION
**Pull request by faergeek**

### CODE REVIEW

File: src/fee/utils.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import { DEFAULT_FEE_CONFIG } from '../constants';
+import { Message } from 'ui/components/transactions/BaseTransaction';
+import { AssetDetail } from 'ui/services/Background';
+import { AccountBalance, AssetBalance } from 'ui/reducers/updateState';
+import { SPONSORED_FEE_TX_TYPES } from './constants';
+
+export function convertFeeToAsset(fee: Money, asset: Asset) {
+  const minSponsoredFee = (asset: Asset) =>
+    asset.id === 'WAVES'
+      ? DEFAULT_FEE_CONFIG.calculate_fee_rules.default.fee
```
Comment (ehassed3) [inline src/fee/utils.ts:16]: I think we need to use the config from https://raw.githubusercontent.com/Waves-Keeper/configs/master/fee.json. If we want to change the value of the field, then there will be a conflict between these two configs.

File: src/controllers/MessageController.js [hunk 2]
```diff
@@ -698,14 +696,14 @@ export class MessageController extends EventEmitter {
           message.account
         );
 
-        const matcherFeeData = !hasFee
-          ? await this._getFee(message, result.data)
-          : {};
+        const matcherFee =
+          message.data.data?.matcherFee ||
```
Comment (ehassed3) [inline src/controllers/MessageController.js:700]: `?.` will it work here?

Comment (faergeek) [reply src/controllers/MessageController.js:700]: Why not? It didn't work earlier because of old babel. Now it works

File: src/fee/utils.ts (hunk 1 above)
Comment (faergeek) [reply src/fee/utils.ts]: Moved fee config to remote config controller

**Review by smelnikov — APPROVED** [review]

**Review by ehassed3 — APPROVED** [review]

**Pull request merged**